blockchain/blocks/src/header.rs

```diff
@@ -268,11 +268,11 @@ impl BlockHeader {
         let signature = self
             .signature()
             .as_ref()
-            .ok_or(Error::InvalidSignature("Signature is nil in header"))?;
+            .ok_or_else(|| Error::InvalidSignature("Signature is nil in header".to_owned()))?;
```

Comment:
There's nothing more beautiful in the world than a big fat method chain

Reply:
????????